### PR TITLE
Api doc #84

### PR DESCRIPTION
**Pull request by IlyaP01**

### CODE REVIEW

File: website/api/api.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+`movement` objects contain array `places_ids`.
+
+Returned JSON example:
+```json
+{
+  "title": "Example",
+  "author": "some author",
+  "description": "This is example quest",
+  "tags": ["example", "test"],
+  "password": "12345abc",
```
Comment (BaevDaniil) [inline website/api/api.md:36]: What about public quests?

File: website/api/api.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+  "file_ids": [1],
+  "time_open": "2011-01-01 00:00:00",
+  "time_close": "2022-01-01 00:00:00",
+  "lead_time": "6 months",
+  "start_question_id": 0,
+  "questions": [
+    {
+      "question_id": 0,
+      "question_text": "",
+      "q_type": "start",
```
Comment (BaevDaniil) [inline website/api/api.md:47]: But what is type of qustion there? This is "open", "choice" or "movement"?

File: website/api/api.md [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+      "answer_options_ids": [0],
+      "movements_ids": [],
+      "files_ids": []
```
Comment (BaevDaniil) [inline website/api/api.md:50]: Where is hints_id array?

File: website/api/api.md [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+instead of `q_type_id`**).
+
+`question`, `hint`, `answer` objects also contain `file_ids`
+key with ids of related files, described in array `files`.
+
+`question` objects contain arrays `hints_ids`, 
+`answer_option_ids`, `movements_ids`
+
+`movement` objects contain array `places_ids`.
+
```
Comment (BaevDaniil) [inline website/api/api.md:28]: I would like to see the details of the 'hint' object.

File: website/api/api.md [hunk 5]
```diff
@@ -0,0 +1,158 @@
...
+      "option_id": 0,
+      "option_text": "",
+      "points": 0,
+      "next_question_id": 1
+    }
+  ],
+  "files": [
+    {
+     "f_id": 1,
+      "url_for_file": "www.files.com/files/228",
```
Comment (BaevDaniil) [inline website/api/api.md:72]: Maybe with file's name?

File: website/api/api.md (hunk 2 above)
Comment (IlyaP01) [reply website/api/api.md:47]: Type "start". It is special fictive type. See db description.

File: website/api/api.md (hunk 5 above)
Comment (IlyaP01) [reply website/api/api.md:72]: No

File: website/api/api.md [hunk 6]
```diff
@@ -0,0 +1,186 @@
...
+      "question_text": "first question",
+      "q_type": "choice",
+      "answer_options_ids": [1],
+      "movements_ids": [],
+      "files": [],
+      "hints": [
+        {
+          "hint_id": 1,
+          "hint_text": "hint",
+          "fine": 5
```
Comment (BaevDaniil) [inline website/api/api.md:75]: And "files": []. Look at 29 string

**Review by BaevDaniil — APPROVED** [review]

**Pull request merged**